Chapters.md

```diff
@@ -0,0 +1,45 @@
+# Block Editor Handbook
+
+## Chapters
+
+### Project Overview
+
+### Developer Documentation
+
+#### Introduction to Block Development
+
+#### Block Development Basics
+
+#### Tutorials
+
+#### Block API Reference
+
+#### Filter Reference
+
+#### SlotFills Reference
+
+#### RichText Reference
+
+#### Internationalization
+
+#### Accessibility
+
+#### Feature Flags
+
+#### Theming for the Block Editor
+
+#### Backward Compatibility
+
+#### Backward Compatibility
```

Comment:
This one looks like accidental copy/pasta.

Reply:
Totally, I like my pasta :-) will fix, thanks.